experiment/cherrypicker/server.go

```diff
@@ -53,6 +54,21 @@ type githubClient interface {
 	ListOrgMembers(org string) ([]github.TeamMember, error)
 }
 
+func HelpProvider(enabledRepos []string) (*pluginhelp.PluginHelp, error) {
+	pluginHelp := &pluginhelp.PluginHelp{
+		Description: `The cherrypick plugin is used for cherrypicking PRs across branches. For every successful cherrypick invocation a new PR is opened against the target branch and assigned to the requester.`,
+	}
+	pluginHelp.AddCommand(pluginhelp.Command{
+		Usage:       "/cherrypick [branch]",
+		Description: "Cherrypick a PR to a different branch.",
```

Comment:
You should mention that this command can be used before or after the PR is merged. That is particularly useful functionality and is worth advertising.